Xamarin.Forms.Core/SwipeItems.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace Xamarin.Forms
+{
+	public class SwipeItems : BindableObject, IList<ISwipeItem>
```

Comment:
this being a BO, and not an Element, prevents parenting. and without parenting, there's no possible tree traversal to resolve styles and resources

Reply:
Good point!.